data/config.yaml

```diff
@@ -17,14 +17,14 @@ name: Fedora
 # The string '$RANDOMPKG' will be replaced by a random "idle" package, if
 # it occurs in the text.
 main-instructions: |
-    <div>
+    <p>
         This is a dashboard to track progress of porting
         Fedora packages to Python 3.
-    </div>
-    <div style="padding-top: 7pt;">
+    </p>
+    <p>
         Read me: <a href="$HOWTO_LINK">So you want to contribute?</a><br/>
-    </div>
-    <div style="padding-top: 7pt;">
+    </p>
+    <p>
         If you would like to jump in and help port, do this:
```

Comment:
The change creates a vertical space between this line and the list below. That means we have 4 visual blocks of text in this section instead of 3, which makes it look a bit too visually complex. That in turn might make some people skip reading it. Though that's only my speculation and how I like to format things.